### PR TITLE
Migrations

### LINKED ISSUE #10: Update password reset migration
Every laravel project comes with a password_resets migration. This migration file doesnt contain id nor timestamps.

Create a package migration to update that table in order to make it have both id and timestamps.

### PR DESCRIPTION
**Pull request by cacpmw**
Added the migrations to update password resets table.

Setting up the package

1. Have a laravel project installed
1. Create `/packages/global-devteam/` folder in the root of the project
1. Clone package repo into the newly created folder
1. Go to the root composer.json file and add the following to the PSR-4 section under autoload section:
```
"Globaldevteam\\LaravelApiPasswordReset\\": "packages/global-devteam/laravel-api-password-reset/src/"
```
5. Open app.php file and add the service provider:
`        \Globaldevteam\LaravelApiPasswordReset\ApiPasswordResetServiceProvider::class,`
6. Run composer dumpautoload
7. Run php artisan migrate

Closes #10

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,14 +1,29 @@
...
         {
             "name": "Carlos Albuquerque Carneiro",
             "email": "carloscarneiropmw@gmail.com"
         }
     ],
-    "minimum-stability": "stable",
-    "require": {}
+    "minimum-stability": "dev",
+    "require": {
+        "illuminate/support": "^5.0|^6.0"
```
Comment (beauraines) [inline composer.json]: Do we need to explicitly state OR 6.0?

Wouldn't it suffice to say "^5" because 6 is greater than 5?

File: composer.json [hunk 2]
```diff
@@ -1,14 +1,29 @@
...
+    },
+    "require-dev": {
+
+        "illuminate/support": "^5.8.15|^6.0",
+        "illuminate/database": "^5.8.15|^6.0"
```
Comment (beauraines) [inline composer.json]: Same with these two...

File: composer.json (hunk 1 above)
Comment (cacpmw) [reply composer.json]: @beauraines no we dont. I will remove that

File: composer.json (hunk 2 above)
Comment (cacpmw) [reply composer.json]: removing

**Comment by cacpmw** [comment]
@beauraines I ran it on L6 and it worked!
```
Carlos-Albuquerque-Carneiro:laravelsix carlos$ php artisan migrate
Migration table created successfully.
Migrating: 2014_10_12_000000_create_users_table
Migrated:  2014_10_12_000000_create_users_table (0.08 seconds)
Migrating: 2014_10_12_100000_create_password_resets_table
Migrated:  2014_10_12_100000_create_password_resets_table (0.05 seconds)
Migrating: 2019_08_19_000000_create_failed_jobs_table
Migrated:  2019_08_19_000000_create_failed_jobs_table (0.04 seconds)
Migrating: 2020_01_19_041653_drop_column_on_password_resets_table
Migrated:  2020_01_19_041653_drop_column_on_password_resets_table (0.04 seconds)
Migrating: 2020_01_19_041724_update_password_resets_table
Migrated:  2020_01_19_041724_update_password_resets_table (0.04 seconds)
Carlos-Albuquerque-Carneiro:laravelsix carlos$ 

```
All I had to do was those aforementioned steps.

**Pull request merged**